src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Cache/BlobCacheAttributeTests.cs

```diff
@@ -5,25 +5,25 @@
 using GovUk.Education.ExploreEducationStatistics.Common.Cache.Interfaces;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
 using GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils;
 using Moq;
 using Xunit;
 
 namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Cache
 {
-    public class BlobCacheAttributeTests : IDisposable
+    [Collection(CacheTestFixture.CollectionName)]
+    public class BlobCacheAttributeTests : IClassFixture<CacheTestFixture>, IDisposable
```

Comment:
Not sure that you need `IDisposable` here or not, but it's inconsistent with the other class where you don't declare it.

Reply:
In this case we do - we have a `Dispose` method to do some cleanup after each test case :smile:

Reply:
Oops, I see that now! 🙂